catkin_ws/src/image_rotation/src/ImageAlgorithm.cpp

```diff
@@ -0,0 +1,16 @@
+#include "ImageAlgorithm.h"
+
+void ImageAlgorithm::RotateImage(cv::Mat &image, double angle)
+{
+    // get rotation matrix for rotating the image around its center in pixel coordinates
+    cv::Point2f center((image.cols - 1) / 2.0, (image.rows - 1) / 2.0);
+    cv::Mat rot = cv::getRotationMatrix2D(center, angle, 1.0);
+    // determine bounding rectangle, center not relevant
+    cv::Rect2f bbox = cv::RotatedRect(cv::Point2f(), image.size(), angle).boundingRect2f();
```

Comment:
meaningful object name for bbox

Reply:
I feel bbox is universal naming trend and if you search the keyword in reference to image processing you will get the desired result

Reply:
in BMW adp code base they use _bounding_box_ instead of bbox